File: src/main/java/com/example/javafxprojectusertask/Entities/Commande.java
package com.example.javafxprojectusertask.Entities;

import java.time.LocalDate;

public class Commande {
    private  int id_user;
    private String adresse;
    private int id_cmd;
   private LocalDate date;

    @Override
    public String toString() {
        return "Commande{" +
                "id_cmd=" + id_cmd +
                ", date=" + date +
                ", id_user=" + id_user +
                ", adresse='" + adresse + '\'' +
                '}';
    }


    public Commande(int id_cmd, LocalDate date, int id_user, String adresse) {
        this.id_cmd = id_cmd;
        this.date = date;
        this.id_user = id_user;
        this.adresse = adresse;
    }

    public Commande()
    {

    }



    public int getId_cmd() {
        return id_cmd;
    }

    public void setId_cmd(int id_cmd) {
        this.id_cmd = id_cmd;
    }

    public LocalDate getDatee() {
        return date;
    }

    public void setDate(LocalDate date) {
        this.date =LocalDate.now();

    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }
    }

File: src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CommandeRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: CommandeRepository::class)]
class Commande
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(name:"idc")]
    private ?int $id = null;

    #[ORM\ManyToOne(inversedBy: 'commandes')]
    #[ORM\JoinColumn(name: "id_user",referencedColumnName:"IdUser",nullable: false)]
    private ?User $id_user = null;

    #[ORM\Column(name:"date",type: Types::DATE_MUTABLE)]
    private ?\DateTimeInterface $date = null;

    #[ORM\Column(name:"adresse",length: 255)]
    private ?string $adresse = null;

    #[ORM\OneToMany(targetEntity: Achat::class, mappedBy: 'idc')]
    private Collection $achats;

    public function __construct()
    {
        $this->achats = new ArrayCollection();
    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getIdUser(): ?User
    {
        return $this->id_user;
    }

    public function setIdUser(?User $id_user): static
    {
        $this->id_user = $id_user;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): static
    {
        $this->date = $date;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $addresse): static
    {
        $this->adresse = $addresse;

        return $this;
    }

    /**
     * @return Collection<int, Achat>
     */
    public function getAchats(): Collection
    {
        return $this->achats;
    }

    public function addAchat(Achat $achat): static
    {
        if (!$this->achats->contains($achat)) {
            $this->achats->add($achat);
            $achat->setIdc($this);
        }

        return $this;
    }

    public function removeAchat(Achat $achat): static
    {
        if ($this->achats->removeElement($achat)) {
            // set the owning side to null (unless already changed)
            if ($achat->getIdc() === $this) {
                $achat->setIdc(null);
            }
        }

        return $this;
    }
}
